examples/sveltekit-scoped/README.md

```diff
@@ -0,0 +1,12 @@
+# SvelteKit (release candidate) + UnoCss Vite Plugin (svelte-scoped mode)
+
+- Works with @sveltejs/kit@1.0.0-next.506 (release candidate) and @sveltejs/adapter-auto@1.0.0-next.80
+- Uses PresetUno and PresetIcons
+- Uses `svelte-scoped` mode
+- Uses `extractorSvelte` to be able to use `class:red-bg-200={true}` in components
+
+## Notes
+
+- To use preflights add `<style uno:preflights global></style>` to your root `+layout.svelte`
```

Comment:
Thanks for the addition! Can we also have the docs about the attribute to the Vite plugin? Thanks

Reply:
Yes, I will add that now. I'm assuming you are talking about https://github.com/unocss/unocss/tree/main/packages/vite#sveltekit.

Reply:
Added. How does https://github.com/unocss/unocss/pull/1672/commits/68033cb3c4b1e2533e3cc3ddc273a31254b018c5 look?